src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbolWithAttributes.cs

```diff
@@ -1039,11 +1039,13 @@ private void DecodeInterceptsLocationChecksumBased(DecodeWellKnownAttributeArgum
             switch (referencedToken)
             {
                 case { Parent: SimpleNameSyntax { Parent: MemberAccessExpressionSyntax { Parent: InvocationExpressionSyntax } memberAccess } rhs } when memberAccess.Name == rhs:
+                case { Parent: SimpleNameSyntax { Parent: MemberBindingExpressionSyntax { Parent: InvocationExpressionSyntax } memberBinding } rhs1 } when memberBinding.Name == rhs1:
```

Comment:
>case [](http://example.com/codeflow?start=16&length=4)

Isn't this case redundant given the other `Parent: MemberBindingExpressionSyntax memberBinding` case below?

Reply:
No, the case below is for an error recovery path (the MemberBinding's parent is not an Invocation).